.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+  - 3.6
+
+services:
+  - docker
+
+install:
+  - docker build -t max-places365-cnn .
```

Comment:
this should just be updated to `max-scene-classifier` since the model / repo name was updated.